packages/core/src/server/helper.ts

```diff
@@ -47,6 +49,29 @@ const formatPrefix = (prefix: string | undefined) => {
   return `${hasLeadingSlash ? '' : '/'}${prefix}${hasTailSlash ? '' : '/'}`;
 };
 
+export const getRoutes = (options: { context: InternalContext }): Routes => {
+  return Object.entries(options.context.environments).reduce<Routes>(
+    (prev, [name, context]) => {
```

Comment:
Rename to `environmentContext` to be more clear and avoid conflict.

```suggestion
    (prev, [name, environmentContext]) => {
```